gradle-module/src/main/java/org/example/Main.java

```diff
@@ -2,24 +2,7 @@
 
 public class Main {
 
-    public static final String CONSTANT = "constant";
-
     public static void main(String[] args) {
         System.out.println("Hello world!");
     }
-
-    /*
```

Comment:
[Severity: Medium] The commented-out block of code appears to be unnecessary and should be removed to maintain code cleanliness. If this code is not needed, it can lead to confusion. Rationale: Keeping the codebase clean and free of commented-out code improves readability and maintainability. Reference: The DRY (Don't Repeat Yourself) principle encourages avoiding unnecessary code clutter.